### DISCUSSION TITLE
Changes to static page not being shown on web

### DISCUSSION THREAD

**Discussion by nickthedalton** [community/community#62078] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Our company has a single static page on Github acting as a signpost to our other company websites.

When I make changes to the index.html file this is not reflected on the viewable web page hours after a commit.  
I've tried editing the file in the browser and downloading downloading Github desktop and pushing the changes up from there.

How long does it take for the changes to be updated in viewable content served?

**Comment by leafstrat** [comment]
> GitHub Pages caches static content for up to 10 minutes by default.

Check the actions tab of the repository and see if it actually built successfully.

**Reply by nickthedalton** [reply]
There appear to be no actions or workflows are evident.

I didn't set this up, a previous company employee did.  It is literally one static HTML page and some image/links.  I just need to change some of the text and have the visible page update.

Not used Github before.  The page is visible so does updating the index.html in the master branch not update the viewable page.  Do I need to Upgrade or make this repository public to enable Pages and create a page from the Master repository to update?

**Reply by leafstrat** [reply]
He may have changed how the build is triggered. If it needs certain thresholds to trigger the build.
I would suggest remaking it fresh if that doesn't sound too intimidating. 

Check these pages for more information:
https://github.com/[ACCOUNT_USERNAME]/[SITE_REPOSITORY]/settings/pages
https://github.com/[ACCOUNT_USERNAME]/[SITE_REPOSITORY]/settings/rules

**Comment by youradrien** [comment]
nice
